src/bio/bio_xstream.c

```diff
@@ -917,21 +984,18 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id)
 		spdk_conf_set_as_default(config);
 
 		spdk_env_opts_init(&opts);
+
 		opts.name = "daos";
+		rc = populate_whitelist(&opts);
+		if (rc != 0) {
+			D_FREE(opts.pci_whitelist);
+			goto out;
+		}
 		if (nvme_glb.bd_shm_id != DAOS_NVME_SHMID_NONE)
 			opts.shm_id = nvme_glb.bd_shm_id;
 
-		if (nvme_glb.bd_mem_size != DAOS_NVME_MEM_PRIMARY) {
-			opts.mem_size = nvme_glb.bd_mem_size;
```

Comment:
This shouldn't be removed. I believe @jbrosenz has a PR to remove these log messages, but we should retain the ability to explicitly set the per-ioserver memory allocation.